nodes/apis/schemas/workspaceWebhook.ts

```diff
@@ -1,19 +1,17 @@
 // ref https://github.com/FormX-ai/form-extractor/blob/master/docs/workspace_webhook.md#webhook-payload
 
 import * as z from 'zod';
-import {
-	extractAPIv2FailureResponseSchema,
-	extractAPIv2SuccessResponseSchema,
-} from './extractSync';
+import { documentSuccessSchema, documentErrorSchema } from './document';
 
-// quoted from spec - "Same as the response of GET /v2/extract/job/{job-id} if should_include_extraction_result, otherwise URLOnlyExtractAPIv2SuccessResponse"
-// Hence, here are just duplicates of extractAPIv2 schemas
-// Ignored URLOnlyExtractAPIv2SuccessResponse since should_include_extraction_result flag is always true now, see `subscribeHook` from zapier/app/src/triggers/workspace_extraction.ts
-export const workspaceWebhookSuccessResponseSchema = extractAPIv2SuccessResponseSchema;
+export const workspaceWebhookSuccessResponseSchema = documentSuccessSchema.extend({
+	status: z.literal('ok'),
```

Comment:
@Ngkaokis Could you please also go back to formx-integration and update the schema there? I know it's gonna be a chore but for sanity we'd want to keep both schema in sync. You might want to move zapier's schema code to `$projectRoot/common` to signify other future integrations should use that as well.